content_scripts/vimium_frontend.coffee

```diff
@@ -178,6 +183,7 @@ initializePreDomReady = ->
     currentKeyQueue: (request) ->
       keyQueue = request.keyQueue
       handlerStack.bubbleEvent "registerKeyQueue", { keyQueue: keyQueue }
+    frameFocused: -> # A frame has received the focus.  We don't care, here. The Vomnibar/UI-component cares.
```

Comment:
I'd say drop it then. It took me quite a while to persuade myself that it wasn't being called, which could have been completely avoided :(